[TERM] : Added new term entry in python for ljust method in strings (#3946)
Codecademy/docs: content/python/concepts/strings/terms/ljust/ljust.md

---
Title: '.ljust()'
Description: 'Left-aligns a string with a specified fill character'
Subjects:
  - 'Computer Science'
  - 'Code Foundations'
Tags:
  - 'Strings'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> The **`.ljust()`** method left-aligns a string with a specified fill character

Reviewer: Please add the following points:
- is a method for strings
- returns a left0aligned string of the length given
Feel free to cover other relevant information.

## Syntax

```pseudo

> string.ljust(length,fillchar)

Reviewer: ```suggestion
string.ljust(length, fillchar)
```

```

> The `.ljust()` method is used to left align the string with the width specified. If the specified width is more than the length of the string, then the remaining part of the string is filled with `fillchar`

Reviewer: ```suggestion
- `length`:
- `fillchar`:
```
Explain both these variables that were used in syntax.

## Examples

> The default fillchar is a blank space. The original string is retrieved if the width is less than or equal to the given string length.

Reviewer: ```suggestion
The default `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.
```
Also, move this above the "Example" header.

> ```python
> text = 'Hello Contributors'
> left_aligned_text = text.ljust(20, '!')
> print(left_aligned_text)
> 
> ```

Reviewer: ```suggestion
```python
text = 'Hello Contributors'
left_aligned_text = text.ljust(20, '!')
print(left_aligned_text)
```
```
Removed a blank line.

This code will result in the following output:

```shell

> Hello Contributors!!
> 

Reviewer: ```suggestion
Hello Contributors!!
```
Removed a blank line.

```

> > **Note:** `.ljust()` returns a new string and does not modify the original string and only one specific character can be mentioned to fill the remaining part of the string with fillchar.

Reviewer: ```suggestion
> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with fillchar.
```

## Codebyte Example

The following code is runnable and uses the `.ljust()` method.

```codebyte/python

> exampleString = 'User, Welcome to docs'
> exampleString = exampleString.ljust(22, '!')
> print(exampleString)
> 

Reviewer: ```suggestion
exampleString = 'User, Welcome to docs'
exampleString = exampleString.ljust(22, '!')
print(exampleString)
```
Removed a blank line.

---
Title: '.ljust()'
Description: 'Left-aligns a string with a specified fill character'
Subjects:
  - 'Computer Science'
  - 'Code Foundations'
Tags:
  - 'Strings'
  - 'Methods'
  - 'Functions'
CatalogContent:
  - 'learn-python-3'
  - 'paths/computer-science'
---

> The method **`.ljust()`** adds padding of a specified length to the right of a given string. The user can specify a character to use as the padding. The number of padding characters added is equal to the length specified in the method call minus the length of the string provided.

Reviewer: ```suggestion
The method **`.ljust()`** left aligns a string for the given length of characters. If the length of the string is less than the parameter given, then padding is added to the right of a given string. The user can specify a character to use as the padding. The number of padding characters added is the length of the string provided subtracted from the length specified in the method call.
```

## Syntax

```pseudo
string.ljust(length, fillchar)
```

### Required Parameters

- `string`: The string to be modified with padding.
- `length`: An integer used to determine how many characters of padding will be added. The number of padding characters will be equal to `length` minus the length of `string`.

### Optional Parameters

> - `fillchar`: A character to fill the missing space (to the right of the string). Default is " " (space).

Reviewer 2: ```suggestion
- `fillchar`: A character to fill the missing space (to the right of the string). The default value is " " (space).
```

## Examples

> The default `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.

Reviewer 2: ```suggestion
The default value for the parameter `fillchar` is a blank space. The original string is retrieved if the length is less than or equal to the given string's length.
```
Same suggestion for this, mentioning that it is a parameter will be helpful to the reader.

```python
text = 'Hello Contributors'
left_aligned_text = text.ljust(20, '!')
print(left_aligned_text)
```

This code will result in the following output:

```shell
Hello Contributors!!
```

> > **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with fillchar.

Reviewer 2: ```suggestion
> **Note:** `.ljust()` returns a new string and does not modify the original string. Only one specific character can be mentioned to fill the remaining part of the string with the parameter `fillchar`.
```
Some readers might get confused if they directly read the sentence. I think it'll be a good practice to mention that it is a parameter.
